File: Cli_Core_CS/Cli_Core_CS/Do_Command.cs
namespace Cli_Core_CS
{
    class Do_Command : Do_Abstract
    {
        protected Cli_Module_Base_Script Module_Script;

        public Do_Command(Cli_Module_Base_Script module_script)
        {
            Module_Script = module_script;
        }

        public override void Do()
        {
            bool is_debug;
            Ref_Boolean debug_res = new Ref_Boolean(false);
            Module_Script.Execute_Script_Command(is_debug = false, debug_res);
        }
    }
}

File: Cli_Core_JavaFX/src/cli_core_javafx_test/Do_Command.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package cli_core_javafx_test;

/**
 *
 * @author mike
 */
public class Do_Command extends Do_Abstract {

    protected Cli_Module_Base_Script Module_Script;

    public Do_Command(Cli_Module_Base_Script module_script) {
        Module_Script = module_script;
    }

    @Override
    public void Do() {
        boolean is_debug;
        Ref_Boolean debug_res = new Ref_Boolean(false);
        Module_Script.Execute_Script_Command(is_debug = false, debug_res);
    }
}
